index.html

```diff
@@ -12,26 +12,114 @@
     <title>William Raga's Portfolio</title>
   </head>
   <body>
-    <header>
+    <header class="header-container">
       <a href="" class="white text-normal align-center roboto logo"
         >William Raga</a
       >
-      <nav class="flex-container-d">
-        <i class="fa-solid fa-bars fa-2x white hamburger-menu"></i>
-        <ul class="flex-container-d lists">
-          <li class="roboto white"><a href="" class="white">Portfolio</a></li>
-          <li class="roboto white"><a href="" class="white">About</a></li>
-          <li class="roboto white"><a href="" class="white">Contact</a></li>
-          <li class="roboto white">
-            <a href="" class="white"
+      <div
+        aria-controls="primary-navigation"
+        aria-expanded="false"
+        class="header-container__nav-list-item-burger burger mobile-nav-toggle"
+        role="navigation"
+      >
+        <div class="burger-menu rotate"></div>
+        <div class="burger-menu dissappear"></div>
+        <div class="burger-menu rotate1"></div>
+      </div>
+      <nav class="flex-container-d header-container__nav">
+
+        <!-- <i class="fa-solid fa-bars fa-2x white hamburger-menu"></i> -->
+
+        <ul
+          class="flex-container-d lists header-container__nav-list primary-navigation nav-links"
+          data-visible="false"
+          id="primary-navigation"
+        >
+          <li class="roboto white header-container__nav-list-item">
+            <a href="#" class="white jav-link-class port">Portfolio</a
```

Comment:
- [ ] You have done a great job linking the other two links for the About and Contact sections. Kindly also link the Portfolio to your projects section so that this might not be a "hanging" link.